test/functional/api/vim_spec.lua

```diff
@@ -99,8 +100,9 @@ describe('api', function()
           [[echo nvim_command_output('echo "nested1\nnested2"') | ls]]))
     end)
 
-    it('does not return shell |:!| output', function()
-      eq(':!echo "foo"\r\n', nvim('command_output', [[!echo "foo"]]))
+    it('returns shell |:!| output', function()
+      local win_lf = (uname() == 'Windows' and '\r') or ''
+      eq(':!echo foo\r\n\nfoo'..win_lf..'\n', nvim('command_output', [[!echo foo]]))
```

Comment:
any reason not to use `iswin()`? It saves some milliseconds.

Reply:
None other than copypasta some code that also didn't use `iswin()`